test/utils.py

```diff
@@ -234,21 +250,40 @@ def start(self, connection_test=True,
                 instance within three seconds deadline after port bound (an attempt to connect is
                 made if there is an option connection_test=True that is present by default).
         """
-        popen_obj = subprocess.Popen(["tarantool", self._instance_cfg_file_name], cwd=self._tmpdir)
-        file_with_port_path = str(self._tmpdir) + '/' + self._instance_cfg_file_name + '.port'
-
-        # Waiting 3 seconds for instance configure itself and dump bound port to temp file.
-        deadline = time.time() + 3
-        while time.time() < deadline:
-            if os.path.exists(file_with_port_path) and os.path.getsize(file_with_port_path) > 0:
-                break
-            time.sleep(0.1)
-        else:
-            raise RuntimeError('Could not find a file with an instance bound port or empty file')
+        tnt_start_cmd = [
+            "tarantool",
+            "--name",
+            instance_name,
+            "--config",
+            self._instance_cfg_file_name,
+        ]
```

Comment:
If a user has specified an instance name and a path to the configuration file, then he most likely expects to run under Tarantool 3. So it's better to throw an error if we run with Tarantool 2 rather than shadow the problem and start something strange.

Reply:
Added throwing an error.